Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -314,23 +324,46 @@ void OpenALStream::SoundLoop()
     // many silence samples.  These do not need to be timestretched.
     if (rate > 0.10)
     {
-      soundTouch.setTempo(rate);
-      if (rate > 10)
+      if (use_timestretching)
+      {
+        soundTouch.setTempo(rate);
+      }
+      else
+      {
+        alSourcef(ui_source, AL_PITCH, static_cast<ALfloat>(rate));
```

Comment:
It could add, sure. But OpenAL Soft is also well optimized for that. SoundTouch and OpenAL Soft's latency after pitch-shifting is minimal, and latency is probably nonexistent when using hardware based OpenAL drivers (which also hardware accelerates the effect, but I bet it is already so light on processor usage that it makes no difference).